ydb/core/tx/schemeshard/schemeshard__table_stats_histogram.cpp

```diff
@@ -6,7 +6,6 @@ namespace NKikimr {
 namespace NSchemeShard {
 
 static bool IsIntegerType(NScheme::TTypeInfo type) {
-    // TODO: support pg types
```

Comment:
Generic not-only-integer realization is enough. Tested in SplitShardsWhithPgKey test.

Reply:
Its enough to work but not enough to work well.
Below in the same file `IsIntegerType()` is used to choose between fast and slow branches for split prefix search. And failing to support integer pg types as integer forces `ChooseSplitKeyByHistogram()` to choose the slow pass.

Reply:
Yes, support for this optimization requires the ability to increment a value of integer pg type which is not  currently supported. I'll do it in separate issue: https://github.com/ydb-platform/ydb/issues/11168